### PR TITLE
ag: add example

### PR DESCRIPTION
**Pull request by guilhermeleobas**
Added an example showing how to perform a search on a different path

----

- [x] The page has 8 or fewer examples.
- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/ag.md [hunk 1]
```diff
@@ -25,3 +25,7 @@
 - Find files with a name matching "foo":
 
 `ag -g {{foo}}`
+
+- Find files containing "foo" on directory bar/baz:
+
+`ag foo {{bar/baz}}`
```
Comment (pxgamer) [inline pages/common/ag.md]: Add token syntax around `foo`.

Also, is `baz` a subdirectory in `bar`? I feel like using the standard path syntax may be more readable and consistent.
```suggestion
`ag {{foo}} {{path/to/directory}}`
```

File: pages/common/ag.md [hunk 2]
```diff
@@ -25,3 +25,7 @@
 - Find files with a name matching "foo":
 
 `ag -g {{foo}}`
+
+- Find files containing "foo" on directory bar/baz:
```
Comment (pxgamer) [inline pages/common/ag.md]: Perhaps this would read better?
```suggestion
- Find files containing "foo" in a specific directory:
```

**Comment by guilhermeleobas** [comment]
Hi @pxgamer, thanks for the review. I just applied your suggestions.

File: pages/common/ag.md [hunk 3]
```diff
@@ -25,3 +25,7 @@
 - Find files with a name matching "foo":
 
 `ag -g {{foo}}`
+
+- Find files containing "foo" in a specific directory:
+
+`ag {{foo}} {{path/to/folder}}`
```
Comment (agnivade) [inline pages/common/ag.md]: Please make this the 2nd example.

Comment (guilhermeleobas) [reply pages/common/ag.md]: Hi @agnivade, my changes were already merged into master. Should I open a new PR for this?

Comment (agnivade) [reply pages/common/ag.md]: Yes. Thank you.

**Pull request merged**